src/peft/peft_model.py

```diff
@@ -374,6 +374,7 @@ def from_pretrained(
         is_trainable: bool = False,
         config: Optional[PeftConfig] = None,
         autocast_adapter_dtype: bool = True,
+        ephemeral_transfers: bool = False,
```

Comment:
How about adding this argument also to `load_adapter`? This would come into practice if users want to load multiple adapters at the same time.

Reply:
Yes that could work, however I am wondering if this can add some boilerplate since you would be able to activate that feature either from `load_adapter` or the lora config itself in that case no ?

Reply:
Thinking a bit more about it, I think this is probably fine, so I agree with @BenjaminBossan